src/evergreen/api.py

```diff
@@ -443,6 +453,16 @@ def tests_by_task(self, task_id, status=None, execution=None):
         url = self._create_url('/tasks/{task_id}/tests'.format(task_id=task_id))
         return [Tst(test, self) for test in self._paginate(url, params)]
 
+    def performance_results_by_task(self, task_id):
+        """
+        Get the 'perf.json' performance results for a given task_id
+
+        :param task_id: Id of task to query for.
+        :return: Contents of 'perf.json'
+        """
+        url = self._create_plugin_url('/task/{task_id}/perf'.format(task_id=task_id))
+        return self._paginate(url)
```

Comment:
Would we want to wrap the returned value in an object? That would allow autocomplete in editors and IDEs which makes working with the API a lot nicer.